src/student.js

```diff
@@ -0,0 +1,18 @@
+let id = 1
+
+function setId(newId) {
+  id = newId
+}
+
+class Student {
+  constructor(firstName, lastName, githubUsername, email) {
+    this.id = id
+    this.firstName = firstName
+    this.lastName = lastName
+    this.githubUsername = githubUsername
+    this.email = email
+    setId(id + 1)
```

Comment:
interesting, I wasn't aware that you could call functions just like that inside a constructor!

Reply:
You certainly can! But please see my comment above on this particular use case before you do this kind of thing